client/src/pages/Courses/editables/RegulationTable.js

```diff
@@ -75,30 +83,51 @@ const RegulationTable = ({ name, updateCourse, course, regulationType }) => {
-                        Add {name}
-                    </Text>
-                </Button>
-                {regulationType && regulationType.length > 0 ? (
-                    <Flex p={1} shadow="md" borderWidth="1px" flex="1" rounded="md" justify="space-between" direction="row">
-                        <Text>{regulationType.map((reg) => `${reg} `)}</Text>
-                        <Box>
-                            <IconButton icon="edit" size="sm" onClick={() => handleRegTableEditClick(regulationType)} />
-                            <IconButton icon="delete" size="sm" onClick={() => handleRegTableDeleteClick(regulationType)} />
-                        </Box>
-                    </Flex>
-                ) : (
-                    <></>
-                )}
+                {regulationList && regulationList.length > 0
+                    ? (
+                        <Flex direction="column" bg={c.whiteGrey} p={1} shadow="md" flex="1" rounded="md">
+                            <Flex direction="row" pb="2px">
+                                <Text width="96.5%" textAlign="center" fontWeight="bold" color={c.midnightBlue}>
+                                    {name}
+                                </Text>
+                                <Flex>
+                                    <OptionsMenu onEdit={() => handleRegTableEditClick(regulationList)} onDelete={() => handleRegTableDeleteClick(regulationList)} />
+                                </Flex>
+                            </Flex>
+
+                            <Flex direction="row" pb="5px">
+                                {regulationList.map((reg) => (
+                                    splitRegulation(reg).map((item, idx) => (
+                                        item.toLowerCase() === "and" || item.toLowerCase() === "or"
+                                            ? (
+                                                <Tag size="sm" key={idx} rounded="full" variant="solid" variantColor="gray" mr={1}>
+                                                    {item}
+                                                </Tag>
+                                            ) : (
+                                                <Tag size="sm" key={idx} rounded="full" variant="solid" variantColor="cyan" mr={1}>
+                                                    <TagLabel>{item}</TagLabel>
+                                                </Tag>
+                                            )
+                                    ))
+                                ))}
+                            </Flex>
+                        </Flex>
+
+                    ) : (
+                        <Button
+                            variantColor="blue"
+                            backgroundColor={c.midnightBlue}
+                            onClick={handleOpen}
+                            isDisabled={regulationList && regulationList.length > 0 ? true : false}
```

Comment:
Would you just be able to drop the ? true :false

Reply:
Removed!